src/server/models/database.js

```diff
@@ -59,6 +59,7 @@ async function createSchema() {
 	await Meter.createTable();
 	await Reading.createTable();
 	await Reading.createCompressedReadingsFunction();
+	await Reading.createCompressedGroupsReadingsFunction();
```

Comment:
This doesn't have anything to do with your PR, but is there a way that we could run `npm run createdb` and have it not fail if we already have an existing database, i.e. make the tables/functions we are missing. This is annoying when trying to update things on a server.

Reply:
Can you open an issue for this? I have an idea of how to do it but, as you say, it's not related to this PR.